File: 1752. Check if Array Is Sorted and Rotated/solution.cpp
class Solution {
public:
  bool check(vector<int>& nums) {
    int found = 0;
    for (int index = 0; index < nums.size(); index++){
      if (nums[index] > nums[(index + 1) % nums.size()]){
        found += 1;
      }
      if (found > 1){
        return false;
      }
    }

    return true;
  }
};

File: 1752. Check if Array Is Sorted and Rotated/solution.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
 var check = function(nums) {
  let found = 0;
  for (let index = 0; index < nums.length; index++){
    if (nums[index] > nums[(index + 1) % nums.length]){
      found += 1;
    }
    if (found > 1){
      return false;
    }
  }
  
  return true;
};